theano/tensor/basic.py

```diff
@@ -3055,7 +3055,9 @@ def var(input, axis=None, keepdims=False):
     centered_input = input - mean_input
 
     # return the mean sqr
-    return mean((centered_input ** 2), axis, keepdims=keepdims)
+    v = mean((centered_input ** 2), axis, keepdims=keepdims)
+    v.name = 'var'
+    return v
```

Comment:
What about giving `std` a name as well (some lines below)? (And maybe fix the `std()` docstring, which says "It calls var() and var()".)